apps/expo-go/ios/Exponent/Versioned/Core/UniversalModules/EXScopedModuleRegistryAdapter.m

```diff
@@ -56,7 +56,8 @@ - (EXModuleRegistry *)moduleRegistryForParams:(NSDictionary *)params
   // Override the FileSystem module with custom document and cache directories
   NSString *documentDirectory = params[@"fileSystemDirectories"][@"documentDirectory"];
   NSString *cachesDirectory = params[@"fileSystemDirectories"][@"cachesDirectory"];
-  EXFileSystemLegacyUtilities *fileSystemModule = [[EXFileSystemLegacyUtilities alloc] initWithDocumentDirectory:documentDirectory cachesDirectory:cachesDirectory];
+  EXFileSystemLegacyUtilities *fileSystemModule = [[EXFileSystemLegacyUtilities alloc] initWithDocumentDirectory:documentDirectory cachesDirectory:cachesDirectory applicationSupportDirectory:@""];
```

Comment:
Breaking change from #29184. Does expo go still use updates? cc @douglowder

Reply:
This was to fix #29058 . It is not an Expo Go specific change, as far as I know.

Reply:
No, but expo go creates this module and the initializer has changed

Reply:
Yes, I believe it would be safest for Expo Go to create the correct path itself, in the same way that line 38 in `FileSystemLegacyUtilities.swift` does.